downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -108,71 +103,83 @@ The return types of the `lang.value:fromJsonFloatString` and `lang.value:fromJso
 
 #### Bug Fixes
 
-To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha4](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+### Breaking Changes
```

Comment:
Why is breaking changes here? The ToC indicates that it's at the end.

Reply:
That was updated as per https://github.com/ballerina-platform/ballerina-dev-website/pull/2093#discussion_r606619298. The TOC is also updated to no longer have the Breaking Changes as a separate topic.

Reply:
To me, this doesn't seem to be correct. It is a level 3 heading similar to `Language updates`, `Runtime updates ` etc. So Is this a sub-topic under `Language updates`? In that case, it should be a level 4 heading.

Reply:
Updated.